### DISCUSSION TITLE
Windows binaries?

### DISCUSSION THREAD

**Discussion by AndrewNewb** [OpenRadioss/OpenRadioss#434] [Q&A]
Hi All,

Great news that Radioss is now open source, well done and good luck to all the developers - I hope your work gets the appreciation it deserves!

I have a probably predictable and tedious question - do you have any plans to release MS Windows binaries? There are many of us who are unable to easily access linux workstations (I do know about VirtualBox and similar but these are often not viable options either).

Thank you in advance for your time.

Cheers

Andrew

**Comment by MarianBulla** [comment]
Hi Andrew, 
yes we are planning to release a MS Windows binary as well as the source code making you able to build it directly under Windwos.
For now, we have the workaround describbed in the Tutorial (for this, please visit the website www.openradioss.org): Using WSL version 2, works well. (I'm using windows 10, with WSL UBUNTU 20.04.4 LTS, and it's working well). 

You can get it directly from the Microsoft Store app, and it's pretty easy to be installed.

Hope I could help you ?

Thanks and best regards,
Marian
